client/components/thank-you/style.scss

```diff
@@ -0,0 +1,8 @@
+.layout__content {
+	padding-top: 0;
+	padding-bottom: 0;
+}
```

Comment:
This can't be added as-is -- it will apply to `.layout_content` _everywhere_ in Calypso!

Reply:
I removed this.